src/pages/index.js

```diff
@@ -1,6 +1,7 @@
 import React from 'react';
 import Helmet from 'react-helmet';
 import App from '../components/app';
+import CreateReactClass from 'create-react-class';
```

Comment:
Not seeing this used anywhere. Could we drop it?

Reply:
Ah yeah, so it is a requirement for the `ReactSlider`. I'm not directly using it anywhere, but the slider seems to need it.